src/Symfony/Component/Finder/Finder.php

```diff
@@ -397,13 +397,17 @@ public function sort(\Closure $closure)
      *
      * This can be slow as all the matching files and directories must be retrieved for comparison.
      *
+     * @param bool $naturalOrdering Whether to use natural ordering or not
```

Comment:
`@param bool $useNaturalOrdering Whether to use natural ordering or not, disabled by default`